CONTRIBUTING.md

```diff
@@ -22,6 +22,13 @@ All new features and bug fixes should be submitted as pull requests, so the comm
 
 When you submit a pull request, @mention a few people you’d like to help you review it. Once those people have signed off on it, the pull request can be merged! Core Team members will handle the merge itself.
 
+## Git Worflow
+
+Foundation uses a git workflow close to [GitFlow](http://nvie.com/posts/a-successful-git-branching-model/). The workflow relies on three branches:
+- **`master`**: The stable branch. It only contains the latest stable version released. Pull requests for docs improvements can be opened on it.
+- **`develop`**: The developing branch used to prepare the next version. It always contains all the new features and bug fixes previously made, and constitutes the more up-to-date version of Foundation. Almost every pull request should be open on this branch. When a new version is released, it is merged on `master`, a support branch is created, and `develop` now target the next version.
```

Comment:
Should we highlight that these are major/minor versions? To make clear that 'version' doesn't refer to patch versions...

Also, English feedback: "more up-to-date" => "most up-to-date". "now target the next" => "now targets the next"